lib/experiments/index.js

```diff
@@ -1,6 +1,12 @@
 'use strict';
 
-const availableExperiments = Object.freeze(['PACKAGER', 'MODULE_UNIFICATION', 'DELAYED_TRANSPILATION', 'SYSTEM_TEMP']);
+const availableExperiments = Object.freeze([
+  'PACKAGER',
+  'MODULE_UNIFICATION',
+  'DELAYED_TRANSPILATION',
+  'SYSTEM_TEMP',
+  'BROCCOLI_WATCHER',
```

Comment:
Each experiment should be tested independently, adding this here also requires adding a corresponding `.travis.yml` entry.

Reply:
I pushed 1e11fb6 to address this.